specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessmentMetadata.json

```diff
@@ -495,16 +496,42 @@
               {
                 "value": "CustomerManaged",
                 "description": "User assessments pushed directly by the user or other third party to Azure Security Center"
+              },
+              {
+                "value": "VerifiedPartner",
+                "description": "An assessment that was created by a verified 3rd party if the user connected it to ASC"
               }
             ]
           }
+        },
+        "partnerData": {
+          "$ref": "#/definitions/SecurityAssessmentMetadataPartnerData"
         }
       },
       "required": [
         "displayName",
         "severity",
         "assessmentType"
       ]
+    },
+    "SecurityAssessmentMetadataPartnerData": {
+      "type": "object",
+      "description": "Describes the partner that created the assessment",
+      "properties": {
+        "partnerName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Name of the company of the partner"
+        },
+        "productName": {
+          "type": "string",
+          "description": "Name of the product of the partner that created the assessment"
+        },
+        "secret": {
+          "type": "string",
+          "description": "Secret to authenticate the partner and verify it created the assessment"
```

Comment:
readonly as well ?

Reply:
This is a write only property, which is not supported for swagger 2.0 (only 3.0)
added to the description instead